src/components/find-block/FindBlock.jsx

```diff
@@ -1,25 +1,25 @@
 import { useCallback, useState } from 'react'
 import { useNavigate } from 'react-router'
-import { useTranslation } from 'react-i18next'
 import { Link } from 'react-router-dom'
-
-import TitleBlock from '~/components/title-block/TitleBlock'
+import { useTranslation } from 'react-i18next'
 import SearchIcon from '@mui/icons-material/Search'
 
 import useBreakpoints from '~/hooks/use-breakpoints'
-import { styles } from '~/containers/student-home-page/find-tutor-block/find-tutor-block.styles'
-import bag from '~/assets/img/student-home/bag.png'
-import { translationKey } from '~/containers/student-home-page/find-tutor-block/constants'
-import { authRoutes } from '~/router/constants/authRoutes'
+import TitleBlock from '~/components/title-block/TitleBlock'
 import InputWithIcon from '~/components/input-with-icon/InputWithIcon'
 import AppButton from '~/components/app-button/AppButton'
 
-const FindTutorBlock = () => {
+import bag from '~/assets/img/student-home/bag.png'
+import { authRoutes } from '~/router/constants/authRoutes'
+import { styles } from '~/components/find-block/find-block.styles'
+
+const FindBlock = ({ translationKey }) => {
   const [inputValue, setInputValue] = useState('')
   const { t } = useTranslation()
   const navigate = useNavigate()
   const { isMobile } = useBreakpoints()
-  const findOffers = `${authRoutes.findOffers.path}?search=${inputValue}`
+  const path = authRoutes.findOffers.path
```

Comment:
if u use it only once it doesn`t need variable

Reply:
> if u use it only once it doesn`t need variable

you are right, removed it